### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IvanMayor**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ services:
     ports:
       - "5432:5432"
     env_file:
-      - .env
\ No newline at end of file
+      - .env.docker
```
Comment (ArsenPidhoretskyi) [inline docker-compose.yml:27]: Add empty line

File: docker-compose.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ services:
              python manage.py migrate &&
              python manage.py runserver 0.0.0.0:8080"
     env_file:
-      - .env
+      - .env.docker
```
Comment (ArsenPidhoretskyi) [inline docker-compose.yml:18]: No need to rename. Just use .env

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Please restore changes in files which we not need to change. It is hard to review code

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
28 files changed. Why you change for example migrations? 
Also why you change api/cinema folder
Please restore redundant change code

**Comment by ArsenPidhoretskyi** [comment]
You change 28 files. Part of them does not need to be changed.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Why have migrations been deleted?

**Pull request closed without merging**